Z/zimg/build_tarballs.jl

```diff
@@ -35,4 +35,5 @@ dependencies = Dependency[
 ]
 
 # Build the tarballs, and possibly a `build.jl` as well.
-build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat="1.6")
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies;
+               julia_compat="1.6", preferred_gcc_version=v"6")
```

Comment:
```suggestion
               julia_compat="1.6", preferred_gcc_version=v"6", clang_use_lld=false)
```

(Apple builds are failing)